lib/utils/common-flags.ts

```diff
@@ -87,3 +90,42 @@ export const json: IBooleanFlag<boolean> = flags.boolean({
 	description: 'produce JSON output instead of tabular output',
 	default: false,
 });
+
+export const dataOutputFlags: flags.Input<DataOutputOptions> = {
+	fields: flags.string({
+		exclusive: ['extended'],
+		description: 'only show provided fields (comma-separated)',
+	}),
+	json: flags.boolean({
+		exclusive: ['no-truncate'],
+		description: 'output in json format',
+		default: false,
+	}),
+	// Not yet supported.
+	// verbose: flags.boolean({
```

Comment:
Maybe we can actually drop `--verbose` in v13. Isn't just a poor man's alternative to `--fields` + `--filter`? (I have not looked at every occurrence of `--verbose`.)

Reply:
Yeah, sounds good!